splunk_topology/conf.yaml.example

```diff
@@ -42,6 +42,7 @@ instances:
     # snapshot: true
     # polling_interval_seconds: 300
     # saved_searches_parallel: 5
+    ignore_saved_search_errors: true
```

Comment:
Can we explain the setting in https://docs.stackstate.com/integrations/splunk_topology/ as well?

Reply:
I can add this but not sure how to modify the doc.

Reply:
modified the doc.